.travis.yml

```diff
@@ -1,18 +1,30 @@
 
-language: java
+language: java C++
 jdk:
   - oraclejdk8
 
 cache:
   directories:
   - $HOME/.m2
 
-
 # Reconstruct the gpg keys to sign the artifacts
 before_deploy:
   - echo $GPG_SECRET_KEYS | base64 --decode | $GPG_EXECUTABLE --import --batch || true
   - echo $GPG_OWNERTRUST | base64 --decode | $GPG_EXECUTABLE --import-ownertrust --batch || true
 
+install:
+  - sudo apt-get install -y cmake libssl-dev libcurl4-openssl-dev liblog4cxx10-dev libprotobuf-dev libboost1.55-all-dev libgtest-dev
+  - pushd $HOME/ && wget https://github.com/google/protobuf/releases/download/v2.6.1/protobuf-2.6.1.tar.gz && popd
```

Comment:
It might also be good to cache both the tgz and the compiled stuff in travis, see line 6.